api/src/request.rs

```diff
@@ -0,0 +1,60 @@
+use ethers::types::Bytes;
+use log::{error, info};
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+
+#[allow(non_snake_case)]
+#[derive(Serialize, Deserialize)]
+/// Data to be sent to the platform API with an offchain request.
+pub struct OffchainInput {
+    /// The chain id of the network to be used.
+    chainId: u32,
+    /// The address of the contract to call.
+    to: Bytes,
+    /// The calldata to be used in the contract call.
+    data: Bytes,
+    /// The Succinct X function id to be called.
+    functionId: Bytes,
```

Comment:
H256

Reply:
`H256` and `Address` serialize in a weird way

Ex. for `H256` the `to_string` is 0xaa...bb, instead of 0xaaaabb

Reply:
Can't you customize the serialization yourself to solve this?